lib/OauthPhirehose.php

```diff
@@ -28,9 +28,9 @@ abstract class OauthPhirehose extends Phirehose
     /**
     */
 	protected function prepareParameters($method = null, $url = null,
-		array $params)
+		array $params= [])
 	{
-		if (empty($method) || empty($url))
+		if (empty($method) || empty($url) )
```

Comment:
Is the space here intentional? I know it's not critical, but just avoiding unnecessary code churn if we can :)

Reply:
Hi - no sorry, I added an "  || empty($params) " thinking that params had to be provided and then it broke so I removed it and looks like i missed a space removal.  (new to doing pull requests , sorry!)

Reply:
@tonybyng Not at all, thanks so much for the PR - looks great!

Merging now :)